server/push/fcm/payload.go

```diff
@@ -262,6 +262,8 @@ func PrepareNotifications(rcpt *push.Receipt, config *AndroidConfig) []MessageDa
 		}
 	}
 
+	titleIOS := "New message"
+	bodyIOS := data["content"]
```

Comment:
It would be nice to do something smarter than hardcoding, but I suppose it's OK for now.

Reply:
Added a TODO for myself.